.gitignore

```diff
@@ -1,7 +1,5 @@
 /func
 /func_*
-/templates/go/events/go.sum
-/templates/go/http/go.sum
```

Comment:
This'll break pkger AFAIK. It'll recognize these as packages now and the directory structure of it will change.

Reply:
I ran `./hack/update-pkger.sh` and committed the result.  I do not know why there was a problem before.  I can only guess it was a file permissions error?

Reply:
It wasn't no. The template tests would fail with a `file not found`. Are we sure we still run those? It's a bit hard to tell because of all the Github Action changes.

Reply:
Yes, part of fixing the errors ended up being a byproduct of cleaning up the actions.
See `test-unit.yaml`, the "Func Unit Tests" for where it runs both core unit tests as well as template unit tests.

Reply:
So when I run `pkger list` with this checked out, here's the result of that

```
knative.dev/kn-plugin-func
 > function:/
 > function:/README.md
 > function:/go.mod
 > function:/go.sum
 > function:/handle.go
 > function:/handle_test.go
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http/README.md
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http/go.mod
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http/go.sum
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http/handle.go
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http/handle_test.go
 > function:/home/mthoemme/go/src/knative.dev/kn-plugin-func/templates/go/http/manifest.yaml
 > function:/manifest.yaml
 > knative.dev/kn-plugin-func:/templates
 > knative.dev/kn-plugin-func:/templates/README.md
 > knative.dev/kn-plugin-func:/templates/go
 > knative.dev/kn-plugin-func:/templates/node
 > knative.dev/kn-plugin-func:/templates/node/events
 > knative.dev/kn-plugin-func:/templates/node/events/README.md
 > knative.dev/kn-plugin-func:/templates/node/events/index.js
 > knative.dev/kn-plugin-func:/templates/node/events/manifest.yaml
 > knative.dev/kn-plugin-func:/templates/node/events/package-lock.json
 > knative.dev/kn-plugin-func:/templates/node/events/package.json
 > knative.dev/kn-plugin-func:/templates/node/events/test
 > knative.dev/kn-plugin-func:/templates/node/events/test/integration.js
 > knative.dev/kn-plugin-func:/templates/node/events/test/unit.js
 > knative.dev/kn-plugin-func:/templates/node/http
...
```

At HEAD, it results in 

```
knative.dev/kn-plugin-func
 > knative.dev/kn-plugin-func:/templates
 > knative.dev/kn-plugin-func:/templates/README.md
 > knative.dev/kn-plugin-func:/templates/go
 > knative.dev/kn-plugin-func:/templates/go/events
 > knative.dev/kn-plugin-func:/templates/go/events/README.md
 > knative.dev/kn-plugin-func:/templates/go/events/go.mod
 > knative.dev/kn-plugin-func:/templates/go/events/handle.go
 > knative.dev/kn-plugin-func:/templates/go/events/handle_test.go
 > knative.dev/kn-plugin-func:/templates/go/events/manifest.yaml
 > knative.dev/kn-plugin-func:/templates/go/http
 > knative.dev/kn-plugin-func:/templates/go/http/README.md
 > knative.dev/kn-plugin-func:/templates/go/http/go.mod
 > knative.dev/kn-plugin-func:/templates/go/http/handle.go
 > knative.dev/kn-plugin-func:/templates/go/http/handle_test.go
 > knative.dev/kn-plugin-func:/templates/go/http/manifest.yaml
 > knative.dev/kn-plugin-func:/templates/node
 > knative.dev/kn-plugin-func:/templates/node/events
 > knative.dev/kn-plugin-func:/templates/node/events/README.md
 > knative.dev/kn-plugin-func:/templates/node/events/index.js
 > knative.dev/kn-plugin-func:/templates/node/events/manifest.yaml
 > knative.dev/kn-plugin-func:/templates/node/events/package-lock.json
 > knative.dev/kn-plugin-func:/templates/node/events/package.json
 > knative.dev/kn-plugin-func:/templates/node/events/test
 > knative.dev/kn-plugin-func:/templates/node/events/test/integration.js
 > knative.dev/kn-plugin-func:/templates/node/events/test/unit.js
 > knative.dev/kn-plugin-func:/templates/node/http
 > knative.dev/kn-plugin-func:/templates/node/http/README.md
 > knative.dev/kn-plugin-func:/templates/node/http/index.js
 > knative.dev/kn-plugin-func:/templates/node/http/manifest.yaml
 > knative.dev/kn-plugin-func:/templates/node/http/package-lock.json
 > knative.dev/kn-plugin-func:/templates/node/http/package.json
 > knative.dev/kn-plugin-func:/templates/node/http/test
 > knative.dev/kn-plugin-func:/templates/node/http/test/integration.js
 > knative.dev/kn-plugin-func:/templates/node/http/test/unit.js
...
```

Note how the paths for the Golang templates have changed. That's what caused the troubles in my tests. When I run `update-pkger.sh` now, I do get a diff as well, though it's sadly hard to introspect what the diff actually is :cry: